kratos/utilities/geometry_tester.h

```diff
@@ -130,7 +130,7 @@ class GeometryTesterUtility
         if(TestHexahedraInterface3D8N(model_part) == false) succesful=false;
 
         if(TestPrism3D6N(model_part) == false) succesful=false;
-//        if(TestPrism3D15N( error_msg ) == false) succesful=false;
+//         if(TestPrism3D15N(model_part) == false) succesful=false;
```

Comment:
```suggestion
         if(TestPrism3D15N(model_part) == false) succesful=false;
```
This should work now